src/components/board/Board.jsx

```diff
@@ -1,3 +1,56 @@
-const Board = () => {};
+import Image from "next/image";
+const Board = ({ name, role, yearMajor, funFact, picture }) => {
+  return (
+    <div className="container w-60 h-90 bg-biscuits-yellow shadow-[20px_-15px_0px_rgb(190,156,106)]">
+      <div className="justify-center">
+        <Image
+          src={picture}
+          alt="picture"
+          className="aspect-square object-scale-down object-contain h-40 w-64"
```

Comment:
```suggestion
          className="aspect-square object-scale-down object-contain h-40 w-64"
```

what does object scale and object contain do?